functions/index.js

```diff
@@ -0,0 +1,122 @@
+const { onSchedule } = require("firebase-functions/v2/scheduler");
+const { setGlobalOptions } = require("firebase-functions/v2");
+const { initializeApp } = require("firebase-admin/app");
+const { getFirestore } = require("firebase-admin/firestore");
+const logger = require("firebase-functions/logger");
+
+initializeApp();
+
+setGlobalOptions({
+    maxInstances: 10,
+    timeoutSeconds: 60,
+    memory: "256MiB"
+});
+
+exports.checkCompletedLessons = onSchedule("* * * * *", async (event) => {
```

Comment:
I am not sure I understand this file, its the functions for firebase functions right?